pyramid_closure/closure/depswriter.py

```diff
@@ -30,8 +30,8 @@
 import shlex
 import sys
 
-import source
-import treescan
+from . import source
+from . import treescan
```

Comment:
Ah sorry, I didn't pay too much attention to the changes 2to3 did. But yes, I also prefer absolute imports, I changed it to:

    from pyramid_closure.closure import source
    from pyramid_closure.closure import treescan